src/test/java/io/confluent/idesidecar/restapi/testutil/FakeSideCarProfile.java

```diff
@@ -0,0 +1,22 @@
+package io.confluent.idesidecar.restapi.testutil;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class FakeSideCarProfile implements QuarkusTestProfile {
+
+  /**
+   * Disable the access token filter to protest unit tests.
+   */
+  @Override
+  public Map<String, String> getConfigOverrides() {
+    return Map.of(
+        "vscode.version", "0.4.0 override",
+        "os.name", "linux-override",
+        "os.arch", "amd64-override",
+        "os.version", "5.11.0-27-azure-override",
+        "version", "0.105.0 override",
+        "vscode.extension.version", "0.21.3 override"
+    );
```

Comment:
Let's use some real values here instead as follows:

```suggestion
  public Map<String, String> getConfigOverrides() {
    return Map.of(
        "vscode.version", "20.1.2",
        "os.name", "GNU/Linux",
        "os.arch", "amd64",
        "os.version", "5.15.0-60-generic",
        "quarkus.application.version", "1.2.3",
        "vscode.extension.version", "0.21.0"
    );
```

Notice that I've renamed `version` -> `quarkus.application.version` since that's the property we lookup in the `SidecarInfo` class.